### PR TITLE
Visualizations

### PR DESCRIPTION
**Pull request by dhemath2**

### CODE REVIEW

File: benchmarks/benchmark_stuff.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 import ndlib.models.ModelConfig as mc
 import networkx as nx
 import pandas as pd
-import pooch
```
Comment (eliotwrobson) [inline benchmarks/benchmark_stuff.py:11]: Can you add this back? This will be needed for some data retrieval stuff later.

File: pyproject.toml [hunk 2]
```diff
@@ -30,6 +30,7 @@ documentation = "https://eliotwrobson.github.io/CyNetDiff/"
 
 [tool.poetry.dependencies]
 python = ">=3.9"
+jupyter = "^1.0.0"
```
Comment (eliotwrobson) [inline pyproject.toml:33]: Two things
1. It looks like this is the old Jupyter release, the newly maintained one is here: https://pypi.org/project/notebook/
2. Could you put this under the benchmark dependencies and update the lockfile? This general dependency section is for anyone installing this and we want to keep the total dependencies as minimal as possible.

**Review by eliotwrobson — CHANGES_REQUESTED** [review]
Two small items, looks good! I will merge in the morning after the changes (I need to modify the docs deployment job first).

File: pyproject.toml (hunk 2 above)
Comment (dhemath2) [reply pyproject.toml:33]: I wasn't sure how to update the lockfile

Comment (eliotwrobson) [reply pyproject.toml]: Run `poetry lock --no-update`

File: benchmarks/Visualizations.ipynb
Comment (eliotwrobson) [inline benchmarks/Visualizations.ipynb:1]: Nit: all lowercase names preferred

**Pull request merged**